__tests__/data/moengage_input.json

```diff
@@ -0,0 +1,967 @@
+[
```

Comment:
Better to use 2-space indentation to keep it similar to others

Reply:
done